modules/platform/login/nuxeo-platform-login-keycloak/pom.xml

```diff
@@ -58,8 +55,8 @@
     <!-- #############################################################################  KEYCLOAK -->
     <dependency>
       <groupId>org.keycloak</groupId>
-      <artifactId>keycloak-tomcat8-adapter</artifactId>
-      <version>${keycloak.version}</version>
+      <artifactId>keycloak-tomcat-adapter</artifactId>
+      <version>12.0.2</version>
```

Comment:
Should we move the version declaration to the root pom?

Reply:
I'll do that with a few others when we move to dependabot so that all deps are in the toplevel POM.